integration-test/run-it.sh

```diff
@@ -32,18 +32,19 @@ function list_storm_processes() {
 
 list_storm_processes || true
 # increasing swap space so we can run lots of workers
-sudo dd if=/dev/zero of=/swapfile.img bs=8192 count=1M
+sudo dd if=/dev/zero of=/swapfile.img bs=4096 count=1M
```

Comment:
The ubuntu image has a 10GB disk, so 8GB doesn't fit. I didn't see the swap exceed a few hundred megs anyway, so I lowered this a bit.